src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -62,9 +62,9 @@ internal int CurrentPasteOffset
         private string author = "None provided";
         private string description;
         private bool hasUnsavedChanges;
-        private readonly ObservableCollection<NodeModel> nodes;
```

Comment:
Do we need `ObservableCollection` here? I thought, that `List` is faster, isn't it?